Modules/MFT/include/MFT/BasicReadoutHeaderQcTask.h

```diff
@@ -31,6 +31,25 @@ namespace o2::quality_control_modules::mft
 /// \author Guillermo Contreras
 class BasicReadoutHeaderQcTask /*final*/ : public TaskInterface // todo add back the "final" when doxygen is fixed
 {
+  // addapted from ITSFeeTask
```

Comment:
This might deserve to be moved to O2 in the future as it might be used by other piece of software.

Reply:
You are right, I will look into it.